docs/versions/java.md

```diff
@@ -22,6 +22,7 @@ When using the image `itzg/minecraft-server` without a tag, the `latest` image t
 | java17-openj9     | 17           | Debian | OpenJ9      | amd64             |
 | java17-graalvm-ce | 17           | Oracle | GraalVM CE  | amd64,arm64       |
 | java17-alpine     | 17           | Alpine | Hotspot     | amd64             |
+| java19-alpine     | 19           | Alpine | Hotspot     | amd64             |
 | java19            | 19           | Ubuntu | Hotspot     | amd64,arm64,armv7 |

```

Comment:
Bump these to java20 and java20-alpine. And apparently I got the base distro wrong by putting Ubuntu.

Can you also add java19 to the deprecated images list, below?